js/rendererjs/uiManager.js

```diff
@@ -125,20 +127,41 @@ function checkUpdate() {
 }
 
 /**
-* Called at window.onload, waits for siad and the plugin system to finish loading to show the UI
+* Called at window.onload.
+* Waits for siad to load, then loads the plugin system.
 * @function UIManager#init
 */
 function init(callback) {
 	// Initialize plugins
-	ui.plugins = plugins
-	// Wait for the plugin system to load, then call callback
+	let plugins = scanFolder(defaultPluginDirectory)
+
+	// The home plugin should be first in the sidebar.
+	// We probably want a priority system for this instead.
+	plugins = plugins.sort((p1, p2) => {
+		if (getPluginName(p2) === defaultHomePlugin) {
+			return 1
+		}
+		// Sort the rest alphabetically
```

Comment:
I don't think alphabetical ordering is desired. For example, we always want `About` to be last.
How did the old code handle this?